### ISSUE TITLE
Investigate MathJax 3

### ISSUE THREAD

**Issue by dae** [ankitects/help-wanted#19]
Anki currently bundles MathJax 2. MathJax 3 includes a bunch of improvements, but it appears to have changed the way it is configured. We need to investigate if the new version can be hooked into Anki's reviewing system. Some things to think about:

- [ ] Does the existing functionality in reviewer.ts still work? We need to be able to register a function to be called once rendering is done, so we can defer card rendering

- [ ] Can users still customize MathJax with some JS on their cards?

- [ ] Will moving to MathJax 3 break things for existing users? 

Anki does not bundle the entire MathJax distribution, so if all of the above are handled, we'll then need to strip out the unnecessary parts (MathML, unused fonts, etc) and commit the required parts to the tree.

**Comment by hgiesel** [comment]
I had a look into this.

I used [this website](https://mathjax.github.io/MathJax-demos-web/convert-configuration/convert-configuration.html) to convert `qt/aqt_data/web/mathjax/conf.js`. 

Also instead of including `Mathjax.js`, I included `tex-chtml.js` from MathJax 3.
However just by including it, I already get the error:

> MathJax localStorage error: Failed to read the 'localStorage' property from 'Window': Storage is disabled inside 'data:' URLs.

I don't know whether you encountered a similar issue while setting up MathJax 2, however it seems MathJax 3 uses `localStorage` within its logic, which prevents it from being used in the reviewer, or the previewer.
